src/tests/multihost/alltests/test_ns_account_lock.py

```diff
@@ -113,13 +129,13 @@ def test_user_inactivated_locked(multihost):
 
         manage_user_roles(multihost, "uid=foo1", "lock", "account")
         with pytest.raises(Exception):
```

Comment:
The exception is caught on on #L35 so the test will fail even when expecting failed login.

Reply:
With login_success retrning bool You can: 
```
assert not login_success(...), "User foo1 shoulod not nhave been able to log in."
```